app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str):
+    supply = 0
+    buy = 0
+    with open(data_file_name, "r") as input_file:
+        intur_data = input_file.readlines()
```

Comment:
I believe there is a type here
```suggestion
        input_data = input_file.readlines()
```

Reply:
I pressed the wrong button and did not sight